build/Version.props

```diff
@@ -8,7 +8,8 @@
     <MajorMinorVersion>$(VersionMajor).$(VersionMinor)</MajorMinorVersion>
     <CliVersionNoSuffix>$(VersionMajor).$(VersionMinor).$(VersionPatch)</CliVersionNoSuffix>
     <CliVersionPrefix>$(CliVersionNoSuffix)-$(ReleaseSuffix)</CliVersionPrefix>
-    <CliBrandingVersion>$(CliVersionNoSuffix) - $(ReleaseSuffix)</CliBrandingVersion>
+    <CliBrandingVersion Condition=" '$(DropSuffix)' != 'true' ">$(CliVersionNoSuffix) - $(ReleaseSuffix)</CliBrandingVersion>
+    <CliBrandingVersion Condition=" '$(CliBrandingVersion)' == '' ">$(CliVersionNoSuffix)</CliBrandingVersion>
```

Comment:
Do we intend for CliBrandingVersion to be set somewhere with DropSuffix != true? I think this would be easier to follow if both lines were conditioned on DropSuffix.

Reply:
Sure. I can change that condition.